common/client_config_v2.go

```diff
@@ -35,8 +35,9 @@ type ClientConfigV2 struct {
 	RetrieversToEnable []RetrieverType
 
 	// Fields required for validator payload retrieval
-	BLSOperatorStateRetrieverAddr string
-	EigenDAServiceManagerAddr     string
+	BLSOperatorStateRetrieverAddr  string
+	EigenDARegistryCoordinatorAddr string
+	EigenDAServiceManagerAddr      string
```

Comment:
if these are required for validator payload retrieval, why are they not part of the payload retriever config? I completely forget why this is separate, might be good to comment.